### PR TITLE
Added the changelog for the 2.1 release

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

File: CHANGES.yaml [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+      Added support for syntax highlighting in the charm details pages in the
+      charmbrowser when the charm author provides a Github Flavoured Markdown
+      readme file.
+    - >
+      Added the ability to drag uncommitted units between machines in the
+      machine view.
+    - Unit statuses are now also shown in the machine view.
+    - Many UI tweaks and updates.
+    - (FIX) When subordinates are deployed extra empty machines are created.
+    - (FIX) Websockets weren't closed properly when switching models.
```
Comment (bac) [inline CHANGES.yaml]: Your tenses between these FIX items don't match.  I think they should be written from the perspective of the applied fix, i.e.

- When subordinates are deployed extra empty machines are not created.
- Websockets are now closed properly when switching models.
- When a user logs out all cookies are deleted.

File: CHANGES.yaml [hunk 2]
```diff
@@ -1,3 +1,23 @@
+- 2.1.0:
+    - Added Juju 2.0-beta support.
+    - Updated all api calls to support Juju 1.x and 2.x-beta facades.
```
Comment (bac) [inline CHANGES.yaml]: s/api/API/

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4646/
Test PASSed.

**Comment by bac** [comment]
:+1: with fixes.

File: CHANGES.yaml (hunk 2 above)
Comment (hatched) [inline CHANGES.yaml]: fixed

File: CHANGES.yaml [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+    - Added the ability to create and switch between models in Juju 2.0.
+    - Updated terminology to match the Juju 2.0 terminology.
+    - >
+      Created user profile view which shows you your available models, bundles
+      and charms after logging into the charmstore.
```
Comment (kadams54) [inline CHANGES.yaml]: I'm a proponent of Oxford commas ("...available models, bundles, and charms..."), but to each their own.

File: CHANGES.yaml (hunk 1 above)
Comment (hatched) [inline CHANGES.yaml]: fixed

File: CHANGES.yaml [hunk 4]
```diff
@@ -1,3 +1,23 @@
...
+    - >
+      Added support for syntax highlighting in the charm details pages in the
+      charmbrowser when the charm author provides a Github Flavoured Markdown
```
Comment (kadams54) [inline CHANGES.yaml]: The official name is "GitHub Flavored Markdown", per https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown

**Comment by kadams54** [comment]
:+1: for me as well, with nitpicks that may be ignored. Thanks!

**Comment by hatched** [comment]
Thanks all! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4647/
Test PASSed.

**Pull request merged**